Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -240,8 +240,7 @@ public final class ProjectPamphletMainCellViewModel: ProjectPamphletMainCellView
 
     // Tracking
 
-    data
-      .takeWhen(self.readMoreButtonTappedProperty.signal)
+    data.take(first: 1)
```

Comment:
Any reason for `take(first:)`?

Reply:
Yes, the signal emits twice. The data signal is initialised with a merge operation. `take(first:)` was added to prevent the event from sending twice to segment

Reply:
Alright.

Reply:
Could you explain why this block was changed if the event you are working on is not happening here? Were we observing the wrong property for the `projectViewed` event?

Reply:
The `projectViewed` event was previously tracked when `Read more button about the campaign` is tapped, which doesn't seem appropriate for a `page viewed` event. That's why I removed the `.takeWhen(self.readMoreButtonTappedProperty.signal)`. This also affects related tests in `ProjectPamphletMainCellViewModelTests`.